query-node/mappings/src/workingGroups.ts

```diff
@@ -688,6 +691,17 @@ export async function workingGroups_StatusTextChanged({ store, event }: EventCon
   await store.save<StatusTextChangedEvent>(statusTextChangedEvent)
 }
 
+export async function contentWorkingGroups_LeadRemarked({ store, event }: EventContext & StoreContext): Promise<void> {
+  const [metadataByte] = new WorkingGroups.LeadRemarkedEvent(event).params
+
+  const leadRemarkMetadata = deserializeMetadata(LeadRemarked, metadataByte)
+  if (leadRemarkMetadata) {
+    await processLeadRemarked(store, event, leadRemarkMetadata)
+  } else {
+    return invalidMetadata('Unrecognized remarked action')
+  }
```

Comment:
You're using the same functions to process app-related actions in both `LeadRemarked` and `MemberRemarked` handlers, but note that in `MemberRemarked` handler there is a try-catch block around `await processMemberRemark` and here there is none. This means that throwing an error from `processCreateAppMessage` / `processUpdateAppMessage` / `processDeleteAppMessage` has 2 different meanings depending on which extrinsic you're processing:
1. When one of those function throws while `MemberRemarked` is being processed - it signals metaprotocol transaction error (so an error is just stored in `MetaprotocolTransactionStatusEvent`)
2. When one of those function throws while `LeadRemarked` is being processed - it crashes the QN

Also note that here you're not saving any `MetaprotocolTransactionStatusEvent`, so unless there is some additional logic introduced in Atlas, it won't recognize the result of this transaction.

You should keep the implementation here consitent with `members_MemberRemarked`